tests/unit/test_channels.py

```
@@ -1224,3 +1225,472 @@ def test_default_behavior_empty_config(self):
+        from agentwire.channels.slack import SlackBridge, SlackConfig
+        bridge = SlackBridge(SlackConfig(bot_token="x", app_token="y"))
+        bridge._set_dm_session("U_ABC", "custom-session")
+        assert bridge._get_dm_session("U_ABC") == "custom-session"
+
+    def test_get_dm_project_path(self, tmp_path, monkeypatch):
+        monkeypatch.setattr(
+            "agentwire.channels.slack.STATE_FILE", tmp_path / "slack-state.json",
+        )
+        from agentwire.channels.slack import SlackBridge, SlackConfig
+        bridge = SlackBridge(SlackConfig(
+            bot_token="x", app_token="y", channels_dir="/tmp/slack",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "bot_token"

(S106)